hubspot/tasks.py

```diff
@@ -33,13 +34,38 @@ def sync_contact_with_hubspot(profile_id):
     response.raise_for_status()
 
 
+@app.task
+def sync_product_with_hubspot(bootcamp_id):
+    """Send a sync-message to sync a Bootcamp with a hubspot product"""
+    body = make_product_sync_message(bootcamp_id)
+    response = send_hubspot_request("PRODUCT", HUBSPOT_SYNC_URL, "PUT", body=body)
+    response.raise_for_status()
+
+
+@app.task
+def sync_deal_with_hubspot(personal_price_id):
+    """Send a sync-message to sync a personal price with a hubspot deal"""
+    body = make_deal_sync_message(personal_price_id)
+    response = send_hubspot_request("DEAL", HUBSPOT_SYNC_URL, "PUT", body=body)
+    response.raise_for_status()
+
+
+@app.task
+def sync_line_with_hubspot(personal_price_id):
+    """Send a sync-message to sync a personal price with a hubspot line"""
+    body = make_line_sync_message(personal_price_id)
+    response = send_hubspot_request("LINE_ITEM", HUBSPOT_SYNC_URL, "PUT", body=body)
+    response.raise_for_status()
+
+
 @app.task
 def check_hubspot_api_errors():
```

Comment:
@pdpinch this task needs a cron job to run it on some frequency (so that, for example, `retry_invalid_line_associations()` is run)

Reply:
added in `bootcamp/settings.py`.